kuma/api/v1/views.py

```diff
@@ -300,3 +301,22 @@ def bc_signal(request):
         serializer.save()
         return Response(serializer.validated_data, status=status.HTTP_201_CREATED)
     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@never_cache
+@require_GET
```

Comment:
What is the reasoning here? I'm not really *questioning*, I'm trying to learn. For example, what's the harm in using `HEAD`. Like, I could use `HEAD` to query if a user exists like this:

```
▶ curl -I http://mdn.localhost:8000/api/v1/users/doesthisusernameexist
HTTP/1.0 405 Method Not Allowed
```

Reply:
Good point, I think we should allow `HEAD` as well.